INSTALL.rst

```diff
@@ -449,8 +449,8 @@ the following commands.
     (invenio)$ inveniomanage config set CFG_BIBSCHED_PROCESS_USER $USER
     (invenio)$ inveniomanage config set CFG_DATABASE_NAME $BRANCH
     (invenio)$ inveniomanage config set CFG_DATABASE_USER $BRANCH
-    (invenio)$ inveniomanage config set CFG_SITE_URL http://0.0.0.0:4000
-    (invenio)$ inveniomanage config set CFG_SITE_SECURE_URL http://0.0.0.0:4000
+    (invenio)$ inveniomanage config set CFG_SITE_URL http://invenio.example.com
```

Comment:
> If you are planning to develop locally

I would set `http://localhost:4000` for both `CFG_SITE_URL` and `CFG_SITE_SECURE_URL`. It's a guide for developers.